presto-docs/src/main/sphinx/release/release-0.251.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.251
+=============
+
+**Highlights**
+==============
```

Comment:
anything worth highlighting. How about the BigQuery connector and ORC dictionary performance improvement.?

Reply:
I feel that the BigQuery connector is surely worth a callout here